### PR TITLE
Adding sample script for cucumber-js with sdk

### PR DESCRIPTION
**Pull request by karanshah-browserstack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,68 @@
...
+
+- Run `npm run sample-local-test`
+
+- You can access the test execution results, and debugging information such as video recording, network logs on [App Automate dashboard](https://app-automate.browserstack.com/dashboard)
+
+## Integration with other NodeJS frameworks
+
+For other NodeJS frameworks samples, refer to following repositories :
+
+- [WebdriverIO](https://github.com/browserstack/webdriverio-appium-app-browserstack)
```
Comment (sriteja777) [inline README.md:62]: Do we want to add other sample repos as well? playwright, codecept..?

File: android/package-lock.json [hunk 2]
```diff
@@ -0,0 +1,7204 @@
+{
+    "name": "cucumber-js-appium-app-browserstack",
+    "version": "1.0.0",
+    "lockfileVersion": 3,
```
Comment (sriteja777) [inline android/package-lock.json]: https://github.com/browserstack/jest-appium-app-browserstack/pull/1#discussion_r1822115664

File: android/package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "version": "1.0.0",
+    "readme": "Cucumber-JS Appium Integration with [BrowserStack](https://www.browserstack.com)",
+    "description": "Cucumber-JS Appium Integration with BrowserStack",
+    "repository": {
+      "type": "git",
+      "url": "git+https://github.com/browserstack/cucumber-js-appium-app-browserstack.git"
+    },
+    "scripts": {
+      "sample-test": "browserstack-node-sdk cucumber-js features/single.feature",
+      "sample-local-test": "browserstack-node-sdk cucumber-js features/local.feature",
```
Comment (sriteja777) [inline android/package.json]: Can we use npx?

File: README.md [hunk 4]
```diff
@@ -1 +1,68 @@
 # cucumber-js-appium-app-browserstack
+
```
Comment (sriteja777) [inline README.md:2]: Do we want to add browserstack logo like in serenity?

https://github.com/browserstack/serenity-appium-app-browserstack/tree/sample-scripts

File: README.md (hunk 1 above)
Comment (karanshah-browserstack) [reply README.md:62]: Not doing as part of this activity.

File: README.md (hunk 4 above)
Comment (karanshah-browserstack) [reply README.md:2]: Kept it consistent across nodejs repo

**Review by khrithik — APPROVED** [review]

**Review by darshilgada24 — APPROVED** [review]

**Review by darshilgada24 — APPROVED** [review]

**Pull request merged**